src/components/jars/Jar.module.css

```diff
@@ -95,3 +95,7 @@
 .tooltipJarContainer {
   cursor: zoom-in;
 }
+
+.custom_tooltip {
```

Comment:
Really just a NIT: Currently, this is the only css var that would use underscores, right? Is it okay to use camelCase for consistencies sake?

Reply:
Right